tests/functional/fetchGit.sh

```diff
@@ -51,9 +51,7 @@ git -C $repo add differentbranch
 git -C $repo commit -m 'Test2'
 git -C $repo checkout master
 devrev=$(git -C $repo rev-parse devtest)
-out=$(nix eval --impure --raw --expr "builtins.fetchGit { url = file://$repo; rev = \"$devrev\"; }" 2>&1) || status=$?
-[[ $status == 1 ]]
-[[ $out =~ 'Cannot find Git revision' ]]
```

Comment:
The fact that these checks had to be removed indicates that the behavior changed in an incompatible manner.
Proposal to revert these changes: https://github.com/NixOS/nix/pull/9668